blst-from-scratch/src/types/g2.rs

```diff
@@ -75,4 +75,9 @@ impl FsG2 {
     pub fn default() -> Self {
         Self(blst_p2::default())
     }
+
+    pub fn rand() -> Self {
```

Comment:
The build will fail because wasm32 expects this method to have a `rand` feature defined above it, like so: `#[cfg(feature = "rand")]`